proposals/lip-0003.md

```diff
@@ -3,11 +3,12 @@ LIP: 0003
 Title: Uniform ordering of delegates list
 Author: Iker Alustiza <iker@lightcurve.io>
 Discussions-To: https://research.lisk.com/t/uniform-ordering-of-delegates-list/
-Status: Active
+Status: Replaced
 Type: Standards Track
 Created: 2018-08-17
 Updated: 2021-12-01
 Requires: 0022
+Superseded-By: 0057
```

Comment:
Should we be consistent and use either `Replaced` and `Replaced-by` or `Superseded` and `Superseded-by`?

Reply:
I also agree with @ricott1

Reply:
I follow LIP 0001 here and the header there https://github.com/LiskHQ/lips/blob/main/proposals/lip-0001.md#lip-header-preamble. I wouldn't change this now.

Reply:
sounds good, cbr